3Dmol/glviewer.js

```diff
@@ -2265,9 +2265,12 @@ $3Dmol.GLViewer = (function() {
             var mostFrames = 0;
             var modelNum = 0;
             for (var i = 0; i < models.length; i++) {
-                if (models[i].getFrames().length > mostFrames) {
```

Comment:
Let's get rid of a `getFrames` as a public api and replace it with `getNumFrames`.  We shouldn't be exposing these implementation details.  Also, change the viewer to be `getNumFrames` as well.